app/models/integrations/base/episode_set_operations.rb

```diff
@@ -0,0 +1,56 @@
+module Integrations
+  module Base
+    module EpisodeSetOperations
```

Comment:
The root of the episode set manipulation.  The consumption side: Deriving the sets of episodes and filtering them for CRUD sync